### PR TITLE
Add maximum wait status attempts

### PR DESCRIPTION
**Pull request by pooknull**

### CODE REVIEW

File: pkg/transferer/load.go [hunk 1]
```diff
@@ -20,6 +20,8 @@ const (
 	LoadStatusOK
 	LoadStatusWait
 	LoadStatusTerminate
+
+	MaxWaitStatusAttempts int = 10
```
Comment (ollevche) [inline pkg/transferer/load.go:24]: it's not attempts limit, please rename this to something more meaningful

`MaxWaitStatusInSequence` maybe?

Comment (ollevche) [reply pkg/transferer/load.go:24]: Applies to `waitAttemptsCounter` as well

File: pkg/transferer/load.go [hunk 2]
```diff
@@ -109,6 +113,13 @@ func (c *LoadChecker) updateStatus() {
 		status = LoadStatusWait
 		log.Warn().Err(err).Msgf("Error while checking metrics load")
 	}
+	if status == LoadStatusWait {
+		c.waitAttemptsCounter++
```
Comment (ollevche) [inline pkg/transferer/load.go:117]: we want to limit sequential waits so the program doesn't hang if the load is not decreasing at all

in a case, when load status changes every few iterations (wait[5] -> ok[2] -> wait[5] ...) we want to proceed normally

File: pkg/transferer/load.go [hunk 3]
```diff
@@ -109,6 +113,13 @@ func (c *LoadChecker) updateStatus() {
...
+		if c.waitAttemptsCounter > MaxWaitStatusAttempts {
+			log.Debug().Msgf("Reached max WAIT status attempts. Sending TERMINATE status")
```
Comment (ollevche) [inline pkg/transferer/load.go:119]: please, use constants for status keys

File: pkg/transferer/load.go (hunk 1 above)
Comment (ollevche) [reply pkg/transferer/load.go:24]: the latter could be just `waitStatusCounter`

**Review by ollevche — CHANGES_REQUESTED** [review]

File: pkg/transferer/load.go (hunk 1 above)
Comment (pooknull) [reply pkg/transferer/load.go]: Resolved in d2bdbaca363c55fe9de7934d01ac8667233954f2

File: pkg/transferer/load.go (hunk 2 above)
Comment (pooknull) [reply pkg/transferer/load.go]: Resolved in d2bdbaca363c55fe9de7934d01ac8667233954f2

File: pkg/transferer/load.go (hunk 3 above)
Comment (pooknull) [reply pkg/transferer/load.go]: Resolved in d2bdbaca363c55fe9de7934d01ac8667233954f2

**Pull request merged**